ui/components/app/edit-gas-display/index.scss

```diff
@@ -39,35 +26,10 @@
     margin: 20px auto;
   }
 
-  &__advanced-button {
-    display: block;
-    margin: 0 auto;
-    background: transparent;
-    color: var(--color-primary-default);
-    font-weight: bold;
-  }
-
   .advanced-gas-controls {
     margin-top: 20px;
   }
 
-  &__education {
```

Comment:
No className exists for these styles
![Screenshot 2023-07-28 at 3 54 21 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/fe9cfd8e-5748-4e31-beae-23b60b69f4c8)